src/jit/hwintrinsic.cpp

```diff
@@ -3,9 +3,44 @@
 // See the LICENSE file in the project root for more information.
```

Comment:
This file contains all the importation and `HWIntrinsicInfo` logic that can be shared between both architectures.

There are probably a few more places code could be shared with some `ifdefs`, but I haven't looked where.